lib/fluent/plugin/out_http_shadow.rb

```diff
@@ -24,7 +24,7 @@ def initialize
     config_param :username, :string, :default => nil
     config_param :password, :string, :default => nil
     config_param :rate, :integer, :default => 100
-    config_param :rate_per_method_hash, :hash, :default => nil
+    config_param :rate_per_host_hash, :hash, :default => nil
```

Comment:
Ah, it's done by you! https://github.com/quipper/fluent-plugin-http_shadow/pull/1
I'm okay then!